wondrous-app/components/Header/HeaderMemoized.tsx

```diff
@@ -90,9 +91,9 @@ const HeaderMemo = ({ isMobile, onSignInClick, showCreateButton, user }: Props)
 
   return (
     <>
+      <TaskViewModalWatcher />
```

Comment:
i think we shouldnt put this here? this causes a bug on cypress test. now there are two modals open essentially? @vefimov  can you make sure?

Reply:
The TaskViewModalWatcher is not opening on certain pages. Can we place it in a central location so that it can open whenever a page needs it?